README.md

```diff
@@ -20,56 +20,54 @@ Download publish settings file from https://manage.windowsazure.com/publishsetti
 package main
 
 import (
-    "fmt"
+	"encoding/base64"
+	"fmt"
 
-    "github.com/Azure/azure-sdk-for-go/management"
-    "github.com/Azure/azure-sdk-for-go/management/hostedservice"
-    "github.com/Azure/azure-sdk-for-go/management/virtualmachine"
-    "github.com/Azure/azure-sdk-for-go/management/vmutils"
```

Comment:
but the example method itself is part of the vmutils package, I guess... so these references need to be put back manually...?